### PR TITLE
quanitze operation expanded to take const argument

### PR DESCRIPTION
**Pull request by d-smirnov**
This PR adds support of constant value for quantize operation.

https://www.tensorflow.org/lite/performance/quantization_spec#references

QUANTIZE (Requantization)
  Input 0:
    data_type  : int8
    range      : [-128, 127]
    granularity: per-tensor
  Output 0:
    data_type  : int8
    range      : [-128, 127]
    granularity: per-tensor

The deepspeech model has quantize operations with constant values

### CODE REVIEW

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -1850,7 +1850,7 @@ def _test_quantize_dequantize(data):
     # First TFLite quantize op converts float32 tensor to int8 tensor - Qnn quantize.
     # Second TFLite quantize op converts int8 tensor to int8 tensor - Qnn requantize.
     data_in = tf.keras.layers.Input(shape=data.shape[1:])
-    relu = tf.keras.layers.ReLU()(data_in)
+    relu = tf.keras.layers.ReLU()(data)
```
Comment (anijain2305) [inline tests/python/frontend/tflite/test_forward.py:1853]: Why is this changed? I would suggest to add a new test for quantize. This one takes the Input layer of Keras, previous line.

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -2726,7 +2726,13 @@ def convert_quantize(self, op):
         assert len(input_tensors) == 1, "input tensors length should be 1"
         input_tensor = input_tensors[0]
         input_tensor_type_str = self.get_tensor_type_str(input_tensor.tensor.Type())
-        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        if self.has_expr(input_tensor.tensor_idx):
+            in_expr = self.get_expr(input_tensor.tensor_idx)
+        else:
+            in_value = self.get_tensor_value(input_tensor)
```
Comment (anijain2305) [inline python/tvm/relay/frontend/tflite.py:2733]: Is it possible to share the model? I am wondering if parsing of constants is the problem here. We can think of this case just like conv where weights are converted to relay consts and we add a var for them.

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (d-smirnov) [reply tests/python/frontend/tflite/test_forward.py:1853]: The idea is to use data as constant parameter for quantize operation which will be inserted on quantisation step

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (d-smirnov) [reply python/tvm/relay/frontend/tflite.py:2733]: Deepspeech itself is here https://github.com/mozilla/DeepSpeech/releases/tag/v0.7.4
However the model which was used was a quantised version fro internal model zoo. I am not sure that I can share it fully

**Comment by tqchen** [comment]
cc @anijain2305 please followup

File: python/tvm/relay/frontend/tflite.py [hunk 3]
```diff
@@ -2726,7 +2726,13 @@ def convert_quantize(self, op):
         assert len(input_tensors) == 1, "input tensors length should be 1"
         input_tensor = input_tensors[0]
         input_tensor_type_str = self.get_tensor_type_str(input_tensor.tensor.Type())
-        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        if self.has_expr(input_tensor.tensor_idx):
```
Comment (anijain2305) [inline python/tvm/relay/frontend/tflite.py:2730]: Does it make sense to use `self.get_tensor_expr` here?

File: tests/python/frontend/tflite/test_forward.py (hunk 1 above)
Comment (anijain2305) [inline tests/python/frontend/tflite/test_forward.py:1853]: Please add a new test with const_input. Lets keep this test as it is

Comment (d-smirnov) [reply tests/python/frontend/tflite/test_forward.py]: Done

File: python/tvm/relay/frontend/tflite.py (hunk 3 above)
Comment (d-smirnov) [reply python/tvm/relay/frontend/tflite.py]: Replaced

**Comment by anijain2305** [comment]
Thanks for the changes @d-smirnov This is merged!
